### PR TITLE
Create QuizPage 

### PR DESCRIPTION
**Pull request by sallamTanna**
#77 

- Create a new page called **QuizPage** that contains three questions decides whether the applicant can continue filling the application or not.
- The conditions need to be improved, for example redirecting the applicant to the application if his answers met the organizations(Edited by using **yup** for validation)

### CODE REVIEW

File: client/src/pages/QuizPage/index.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        <RadioGroup aria-label="trustees" name="trustees" value={trusteesValue} onChange={handleTrusteesChange}>
+          <FormControlLabel value="trusteesYes" control={<Radio />} label="Yes" />
+          <FormControlLabel value="trusteesNo" control={<Radio />} label="No" />
+        </RadioGroup>
+        <p>3. Are you a charity</p>
+        <RadioGroup aria-label="charity" name="charity" value={charityValue} onChange={handleCharityChange}>
+          <FormControlLabel value="charityYes" control={<Radio />} label="Yes" />
+          <FormControlLabel value="charityNo" control={<Radio />} label="No" />
+        </RadioGroup>
+        <Button className='page-submit' onClick={handleSubmit}>submit</Button>
```
Comment (dubhcait) [inline client/src/pages/QuizPage/index.js]: could we change submit to proceed?

File: client/src/pages/QuizPage/index.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      })
+
+      requiredSchema
+      .isValid({
+        trusteesValue, 
+        charityValue,
+        annualTurnover
+      })
+      .then(function(valid) {
+        //valid is boolean, when valid === true, then go to the QuestionsPage
```
Comment (dubhcait) [inline client/src/pages/QuizPage/index.js]: we can use the handleStaticButton as long as long as we pass it as a prop. Good job.

File: client/src/pages/QuizPage/index.js (hunk 1 above)
Comment (sallamTanna) [reply client/src/pages/QuizPage/index.js]: Yes, edited

**Comment by sallamTanna** [comment]
> It's looking really good.
> I'm getting this in the console
> image
> 
> Is it the same for you?

this is because there was no initial value assigned to useState. Solved by assigning the initial value.

**Pull request merged**